### PR TITLE
Prep for release of @agoric/ertp@0.2.0

### PR DESCRIPTION
**Pull request by katelynsills**
Prep for release of @agoric/ertp@0.2.0

### CODE REVIEW

File: packages/cosmic-swingset/package.json [hunk 1]
```diff
@@ -20,7 +20,7 @@
   "dependencies": {
     "@agoric/bundle-source": "^1.0.4",
     "@agoric/captp": "^1.0.2",
-    "@agoric/ertp": "^0.1.10",
+    "@agoric/ertp": "0.2.0",
```
Comment (michaelfig) [inline packages/cosmic-swingset/package.json:23]: Can you make this `"^0.2.0"`?

File: packages/zoe/package.json [hunk 2]
```diff
@@ -32,7 +32,7 @@
   },
   "homepage": "https://github.com/Agoric/agoric-sdk#readme",
   "dependencies": {
-    "@agoric/ertp": "^0.1.12",
+    "@agoric/ertp": "0.2.0",
```
Comment (michaelfig) [inline packages/zoe/package.json:35]: Same here.

**Review by michaelfig — APPROVED** [review]
Two minor nits.

File: packages/zoe/package.json (hunk 2 above)
Comment (katelynsills) [reply packages/zoe/package.json:35]: Yup, good catch

**Pull request merged**